pkg/controller/route53/hostedzone/controller.go

```diff
@@ -109,31 +125,30 @@ func (e *external) Observe(ctx context.Context, mgd resource.Managed) (managed.E
 	}
 
 	res, err := e.client.GetHostedZoneRequest(&route53.GetHostedZoneInput{
-		Id: aws.String(meta.GetExternalName(cr)),
+		Id: aws.String(fmt.Sprintf("/hostedzone/%s", meta.GetExternalName(cr))),
```

Comment:
any value in making this string format a const?

Reply:
done!

Reply:
Awesome, thanks! My approval still stands so merge as you see fit :)